TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -364,6 +383,32 @@ public void onClick(View view) {
         }
     }
 
+    public void onClickOkLoadNavMapDialog(final String mapName) {
+        if (mapName == null || mapName.isEmpty()) {
+            Log.e(TAG, "Map name is null or empty, unable to load the navigation map");
+            displayToastMessage(R.string.map_name_error);
+            return;
```

Comment:
This error handling is not very user friendly. Is it a valid use case to load a map with an empty map name? Or to press Ok without a map being selected?

- Try to implement the dialogFragment in such a way that the map name is  never empty by e.g. not listing the maps with empty map name.
- Disable the Ok button until a valid map was selected. Otherwise, only allow cancel.